packages/perseus/src/widgets/interactive-graphs/math/angles.ts

```diff
@@ -20,19 +23,55 @@ export function polar(r: number | vec.Vector2, th: number): vec.Vector2 {
     th = (th * Math.PI) / 180;
     return [r[0] * Math.cos(th), r[1] * Math.sin(th)];
 }
-
-export const findAngle = (
-    point1: vec.Vector2,
-    point2: vec.Vector2,
-    vertex?: vec.Vector2,
+// This function calculates the angle between two points and an optional vertex.
+// If the vertex is not provided, the angle is measured between the two points.
+// This does not account for reflex angles or clockwise position.
+export const findAngleFromVertex = (
```

Comment:
There was only one case I could find where we were actually making use of all 3 points for findAngle, which was the polygon graph. 

In order to simplify these functions, I've opted to scope this function to only calculate the angle from a single point and the vertex. I have updated the polygon graph to use the new getClockwiseAngle method below, which is able to be shared in several other places.